src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -51,10 +53,11 @@
     @Test
     void createConsumer(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
         // create consumer
-        consumerService().createConsumer(context, groupId, consumerWithEarliestReset);
+        ConsumerService consumer = consumerService().createConsumer(context, groupId, consumerWithEarliestReset);
```

Comment:
the `consumerService()` return a singleton, it's useless to save it in a variable.